### PR TITLE
LEAN-2361 

### PR DESCRIPTION
**Pull request by hhusban**
LEAN-2361-- TD-080a - Expose versions of services: Pressroom - Implementation

### CODE REVIEW

File: src/PromClientRegistryConfig.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * limitations under the License.
+ */
+import fs from 'fs'
+import client from 'prom-client'
+
+const metricsMap = [
+  {
+    name: 'json_schema_version_info',
+    packageName: 'json-schema',
+    help: 'The json-schema version by package.json',
```
Comment (mnatsheh) [inline src/PromClientRegistryConfig.ts:23]: Let's replace "json-schema" with "@manuscripts/json-schema" in the help. Same for the rest.

File: src/PromClientRegistryConfig.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        labelNames: metric.labelNames,
+      })
+      const processedVersion = processVersion(metric.packageName)
+      gauge
+        .labels(processedVersion.version, ...processedVersion.versionParts)
+        .set(1)
+    }
+  }
+}
+function processVersion(packageName: string): {
```
Comment (mnatsheh) [inline src/PromClientRegistryConfig.ts:54]: Can we use the `semver` package to parse the version and as the return type?

File: src/PromClientRegistryConfig.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  const versionFromNodeModule = getVersion(packageName)
+  const vSplit = versionFromNodeModule.split('.')
+  return {
+    version: versionFromNodeModule,
+    versionParts: [vSplit[0], vSplit[1], vSplit[2]],
+  }
+}
+
+function getVersion(packageName: string): string {
+  const f1 = fs.readFileSync(
```
Comment (mnatsheh) [inline src/PromClientRegistryConfig.ts:67]: Let's use async/await for this (`import { promises as fs } from 'fs'`).

File: src/PromClientRegistryConfig.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs'
+import client from 'prom-client'
+
+const metricsMap = [
```
Comment (mnatsheh) [inline src/PromClientRegistryConfig.ts:19]: Is the API version included by default?

**Review by mnatsheh — CHANGES_REQUESTED** [review]

File: src/PromClientRegistryConfig.ts (hunk 4 above)
Comment (hhusban) [reply src/PromClientRegistryConfig.ts:21]: Yes, under app_version

**Review by mnatsheh — APPROVED** [review]

**Pull request merged**